### DISCUSSION TITLE
Temperature got stuck

### DISCUSSION THREAD

**Discussion by Gunnar2019** [BernhardSchlegel/BierBot-Bricks#31] [Q&A] [answered]
During my first brew I got a problem: the temperature from the sensor didn't raise anymore around 70°C.

I could reproduce the problem at home. The temperature slways got stuck at higher temperatures. I checked the connector. Didn't help.

Next step is a new sensor. Any other ideas?

**Comment by BernhardSchlegel** [comment, marked as answer]
Hey Gunnar,

sorry to hear that. Aside from the broken sensor you already mentioned another reason might be that the Brick lost connection. In an automatic process the last received temperature will then be used. You can check the "Bricks" page, when the last signal was received.

Cheers,
Bernhard.

**Reply by Gunnar2019** [reply]
Thanks Bernhard,

I manual mode I could change the temperature/heating. Just the temperature was stuck. I bought some new DS18B20s and will try this weekend again!

**Reply by Gunnar2019** [reply]
I used a new sensor yesterday. I worked well! Thank you!!
